docs/core/install/linux-debian.md

```diff
@@ -122,7 +122,7 @@ sudo apt-get update; \
 
 ## Dependencies
 
-[!INCLUDE [linux-install-dependencies](includes/linux-install-dependencies.md)]
+[!INCLUDE [linux-rpm-install-dependencies](includes/linux-rpm-install-dependencies.md)]
```

Comment:
The dependency list for Debian is not the same as what is listed in the rpm document. 

Debian 10: libicu63, libssl1.1
Debian 9: libicu57, libssl1.1

Reply:
Fixed